### ISSUE TITLE
Version conflict with precommit

### ISSUE THREAD

**Issue by dreispt** [OCA/pylint-odoo#330]
Can we avoid pinning down specific dependency versions?

See https://github.com/OCA/management-system/pull/376, build log https://travis-ci.com/github/OCA/management-system/jobs/502231662#L307
```
An unexpected error has occurred: CalledProcessError: command: ('/home/travis/.cache/pre-commit/repoy2ix0myc/py_env-python3/bin/python', '-mpip', 'install', '.', 'pylint-odoo==3.0.3')

return code: 1

expected return code: 0

stdout:

    Processing /home/travis/.cache/pre-commit/repoy2ix0myc

    Collecting pylint-odoo==3.0.3

      Downloading pylint-odoo-3.0.3.tar.gz (60 kB)

    Collecting pylint==2.3.1

      Downloading pylint-2.3.1-py3-none-any.whl (765 kB)

    Collecting astroid==2.2.0

      Downloading astroid-2.2.0-py3-none-any.whl (176 kB)

    INFO: pip is looking at multiple versions of pre-commit-dummy-package to determine which version is compatible with other requirements. This could take a while.

    

    The conflict is caused by:

        pre-commit-dummy-package 0.0.0 depends on pylint==2.3.1

        pylint-odoo 3.0.3 depends on pylint==2.3.0

    

    To fix this you could try to:

    1. loosen the range of package versions you've specified

    2. remove package versions to allow pip attempt to solve the dependency conflict

    

    

stderr:

    ERROR: Cannot install pre-commit-dummy-package==0.0.0 and pylint-odoo==3.0.3 because these package versions have conflicting dependencies.

    ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies
```

**Comment by moylop260** [comment]
Sorry for delay @dreispt 

Is it still reproduced?

Do you have the URL of the error in the CI?

**Comment by dreispt** [comment]
It is included in the issue description. But it seems the build worked some days after, so probably nothing to do.
